source/fileUtils.py

```diff
@@ -0,0 +1,16 @@
+import os
```

Comment:
@jcsteh Fixed. There are still quite some files without this header in the codebase though. I just didn't think about it and it might have helped if I saw the header every time while browsing parts of the source.